grafana-plugin/src/utils/decorators.ts

```diff
@@ -0,0 +1,27 @@
+import { LoaderStore } from 'models/loader/loader';
+
+export function AutoLoadingState(actionKey: string) {
```

Comment:
This is newly added, I think it will really enhance UX experience if we start providing feedback when we wait for backend. This is to wrap class methods

Reply:
hmm, why is function name capitalised?